### PR TITLE
Implement disable wallet for Algorand blockchain

### PR DESCRIPTION
**Pull request by Riddlerrr**
https://comakery-upside.atlassian.net/browse/UPS-257

### CODE REVIEW

File: lib/blockchains/AlgorandBlockchain.js [hunk 1]
```diff
@@ -270,8 +296,96 @@ class AlgorandBlockchain {
...
+      const amountWithoutFee = totalAmount - mAlgoFee - 1000000
+      chainTransaction = this.chain.new.Transaction()
+      txn = this.transferAlgoTx(hotWallet.address, coinAddress, amountWithoutFee)
+      action = await this.chain.composeAction(chainjs.ModelsAlgorand.AlgorandChainActionType.Payment, txn)
+      chainTransaction.actions = [action]
+      await chainTransaction.setDesiredFee(mAlgoFee)
+      await chainTransaction.prepareToBeSigned()
+      await chainTransaction.validate()
+      await chainTransaction.sign([chainjs.HelpersAlgorand.toAlgorandPrivateKey(hotWallet.privateKey)])
+      const tx_result = await chainTransaction.send(chainjs.Models.ConfirmType.After001)
```
Comment (Alexey1100) [inline lib/blockchains/AlgorandBlockchain.js:321]: There seem to be a lot of repetitive code.
Why can't we reuse the `sendTransction` method?

File: lib/blockchains/AlgorandBlockchain.js [hunk 2]
```diff
@@ -270,8 +296,96 @@ class AlgorandBlockchain {
...
+          console.log(msg)
+          result.coinsWithdrawalTx = { status: "success", txHash: tx.transactionId, message: msg }
+        } else {
+          const msg = "Error during withdraw Algo tx"
+          console.error(msg)
+          result.coinsWithdrawalTx = { status: "failed", txHash: null, message: msg }
+        }
+
+      } else {
+        result.coinsWithdrawalTx = { status: "skipped", txHash: null, message: "ETH balance is zero, transaction skipped" }
```
Comment (Alexey1100) [inline lib/blockchains/AlgorandBlockchain.js:385]:
```suggestion
        result.coinsWithdrawalTx = { status: "skipped", txHash: null, message: "ALGO balance is <= 1, transaction skipped" }
```

File: lib/blockchains/AlgorandBlockchain.js (hunk 1 above)
Comment (Riddlerrr) [reply lib/blockchains/AlgorandBlockchain.js:321]: `sendTransction` locked on AppNoOp transaction type for now.
But we can change it in Algorand grouped transactions PR

**Review by sofiyareverse — APPROVED** [review]
It seems to work fine

**Review by Alexey1100 — APPROVED** [review]

**Pull request merged**